spec/views/layouts/user_mailer.html.erb_spec.rb

```diff
@@ -35,7 +35,7 @@
     expect(rendered).to have_link(MarketingSite.nice_help_url, href: MarketingSite.help_url)
   end
 
-  it 'includes link to About login.gov' do
+  it 'includes link to About Login.gov' do
     expect(rendered).to have_link(t('mailer.about', app: APP_NAME), href: MarketingSite.base_url)
```

Comment:
we need to update the APP_NAME constant defined here:
https://github.com/18F/identity-idp/blob/1c56089368848432fddbd19e02b8263cea3fddd5/config/application.rb#L17

Reply:
Updated here 94e6f42